biit_server/database.py

```diff
@@ -1,11 +1,20 @@
 from google.cloud import firestore
 
 
-def add(obj):
+def add(obj, database):
     """Helper function to add object into database"""
     return ""
 
 
-def update(id, update_data):
+def update(id, database, update_data):
     """Helper function to update objects in the database."""
     return ""
+
+
+def remove(id, database):
+    """Helper function to remove objects in the database"""
+    return ""
+
+def get(id, database, query=None):
+    """Helper function to query the database"""
+    return ""
```

Comment:
Can we get some better docs here? I know its not implemented yet but firebase is new to everyone so any help on how to use this will help.